README.md

```diff
@@ -3,6 +3,16 @@
 A proposed [WebAssembly System Interface](https://github.com/WebAssembly/WASI)
 API to add native thread support.
 
+> __NOTE__: this proposal is considered a legacy proposal, retained for engines
```

Comment:
Can we perhaps mention there are runtimes already implementing this proposal (WAMR specifically) and teams that rely on it, so it will be around for quite some time? 

I understand we want to encourage people to wait and use the new proposal, but if there are people like us who got stuck with this proposal and can't migrate, they can open a ticket in this repo to contact us for any relevant topic - I'll be actively monitor this repo and reply to those tickets.

Reply:
Yeah, those are good points. I'll try to add something about that. Can you think of anyone else I should put on the "contact these people" list?

Reply:
You can put my name on the list for now. Not sure if anybody from WAMR and the community around it (maybe @yamt , @wenyongh) would like to be included.

Reply:
please feel free to include me.
however, i'm not sure if such a list is necessary.
a list of links to github issues of relevant repos, including this repo, should be enough.
it probably even works better than a person-level contact list.

Reply:
please feel free to include me too if the list is added.

Reply:
Ok, see 21f7eb2; @yamt, I ended up doing both &mdash; your "add an issue" suggestion plus "a list of people" to tag.